nix/modules/devShells.nix

```
@@ -130,6 +130,16 @@
                 cargo generate-lockfile --offline --manifest-path=crates/test_utils/wasm/wasm_workspace/Cargo.toml
               '')
 
+              (pkgs.writeShellScriptBin "scripts-cargo-update" ''
+                 set -xeou pipefail
```

i'm not sure if this is needed to make `-o` understand the next argument is meant for it, but it's at least more readable for me.

```suggestion
                 set -xeu -o pipefail
```

that said, this script won't be able to show the effects of this anyway :laughing: